org.kde.PlatformTheme.QtSNI.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "org.kde.PlatformTheme.QtSNI",
+    "branch": "5.14",
+    "runtime": "org.kde.Platform",
+    "build-extension": true,
+    "sdk": "org.kde.Sdk",
+    "runtime-version": "5.14",
+    "appstream-compose": false,
+    "separate-locales": false,
+    "modules": [
+        {
+            "name": "qtsni",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DQT_PLUGINS_DIR=/usr/share/runtime/lib/plugins/QtSNI"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/ilya-fedin/qtsni-theme",
+                    "tag": "v1.0.0",
+                    "commit": "ce105986951d773c90a10dd814a477e07dff69aa"
+                }
+            ]
+        },
+        {
+            "name": "appdata",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p ${FLATPAK_DEST}/share/appdata",
+                "cp org.kde.PlatformTheme.QtSNI.appdata.xml ${FLATPAK_DEST}/share/appdata",
+                "appstream-compose --basename=org.kde.PlatformTheme.QtSNI --prefix=${FLATPAK_DEST} --origin=flatpak org.kde.PlatformTheme.QtSNI"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "org.kde.PlatformTheme.QtSNI.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Fixed